client/me/concierge/shared/no-available-times.js

```diff
@@ -26,24 +27,29 @@ class NoAvailableTimes extends Component {
 				<PrimaryHeader isWhiteGlove={ isWhiteGlove } />
 				<Card>
 					<h2 className="shared__no-available-times-heading">
-						{ translate( 'Sorry, there are no sessions available' ) }
+						{ translate( 'Sorry, all upcoming sessions are full.' ) }
 					</h2>
-					{ isWhiteGlove && (
-						<>
-							We schedule one-on-one sessions up to 24 hours in advance and all upcoming sessions
-							are full. Please check back later or{ ' ' }
-							<a href="https://wordpress.com/help/contact">contact us in Live Chat</a>.
-						</>
+					{ translate(
+						'We add new sessions daily, so please check back soon for more options. In the meantime, consider attending one of our expert webinars on a wide variety of topics designed to help you build and grow your site. {{externalLink1}}View webinars{{/externalLink1}} or {{externalLink2}}contact us in Live Chat{{/externalLink2}}.',
+						{
+							components: {
+								externalLink1: (
+									<ExternalLinkWithTracking
+										icon={ false }
+										href="https://wordpress.com/webinars/"
```

Comment:
The URL in this format creates an error when using wpcom in a non-EN locale. Try setting your interface language to non-EN and then click on this link. It converts this URL to a localized URL format es.wordpress.com/webinars and since this URL does not exist, it redirects again to wordpress.com/me/help  which is a blank page. The redirect behaviour is a bit strange but to fix this, I suggest making this:

```
href="/webinars"
```